src/components/tx-flow/index.tsx

```diff
@@ -18,7 +20,7 @@ export const TxModalContext = createContext<TxModalContextType>({
 })
 
 export const TxModalProvider = ({ children }: { children: ReactNode }): ReactElement => {
-  const [txFlow, setFlow] = useState<TxModalContextType['txFlow']>(undefined)
+  const [flow, setFlow] = useState<TxModalContextType['txFlow']>(undefined)
```

Comment:
Please avoid changes like this next time.

Reply:
`setTxFlow` accepts `txFlow` as an argument and noticed that this state and the setter aren't similarly named. I've opted to rename the argument instead in 31f019ca951be8f57093fdfda0d388e3a1ba0f65.